### DISCUSSION TITLE
复制粘贴的图片好像不会触发上传接口，能不能支持一下

### DISCUSSION THREAD

**Discussion by Jiangw2020** [Qexo/Qexo#292] [General]
如果不能直接复制网络上的图片粘贴上传到图床，则需要先保存图片到本地

**Comment by am-abudu** [comment]
本项目的编辑器引用 vditor 项目，这个问题我无法给出解决，你需要在原项目提交 
https://github.com/Vanessa219/vditor

**Comment by ning0818** [comment]
`
![](网络图片链接)
`
